src/jrubyTest/java/org/embulk/util/rubytime/TestRubyDateTimeFormatterFormatWithJRuby.java

```diff
@@ -0,0 +1,157 @@
+/*
+ * Copyright 2020 The Embulk project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.embulk.util.rubytime;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import java.time.OffsetDateTime;
+import java.time.ZoneId;
+import java.time.ZoneOffset;
+import java.time.ZonedDateTime;
+import java.util.Arrays;
+import java.util.Locale;
+import java.util.TimeZone;
+import java.util.stream.Stream;
+import org.joda.time.DateTime;
+import org.joda.time.DateTimeZone;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.MethodSource;
+
+/**
+ * Tests formatting by RubyDateTimeFormatter with JRuby's org.jruby.util.RubyDateFormat.
+ *
+ * <p>The main purpose of these tests is to confirm it works almost the same with org.jruby.util.RubyDateFormat.
+ * Embulk's legacy {@code TimestampFormatter} has used {@code org.jruby.util.RubyDateFormat} for a long time.
+ */
+public class TestRubyDateTimeFormatterFormatWithJRuby {
+    @ParameterizedTest
+    @MethodSource("provideZonedDateTime")
+    public void testZonedDateTime(final ZonedDateTime datetime) {
+        assertRubyDateFormat(datetime);
+    }
+
+    @ParameterizedTest
+    @MethodSource("provideOffsetDateTime")
+    public void testOffsetDateTime(final OffsetDateTime datetime) {
+        assertRubyDateFormat(datetime);
+    }
+
+    @SuppressWarnings("deprecation")  // For use of org.jruby.util.RubyDateFormat.
+    private void assertRubyDateFormat(final ZonedDateTime datetime) {
+        final String format = "%Y-%m-%dT%H:%M:%S %Z";
+
+        final org.jruby.util.RubyDateFormat jrubyFormat = new org.jruby.util.RubyDateFormat(format, Locale.ROOT, true);
+        final DateTime jodaDateTime = new DateTime(
+                datetime.toInstant().toEpochMilli(),
+                DateTimeZone.forTimeZone(TimeZone.getTimeZone(datetime.getZone())));
+        jrubyFormat.setDateTime(jodaDateTime);
+        jrubyFormat.setNSec(datetime.getNano() / 1000);
+        final String expected = jrubyFormat.format(null);
+
+        final RubyDateTimeFormatter formatter = RubyDateTimeFormatter.ofPattern(format);
+        final String actual = formatter.formatWithZoneNameStyle(datetime, RubyDateTimeFormatter.ZoneNameStyle.SHORT);
+
+        System.out.println(expected);
```

Comment:
Having `Expected` and `Actual` prefix would make it easier to see the results when many.